### PR TITLE
Adds a new mass hallucination, courtesy of Izzy

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

File: code/modules/events/mass_hallucination.dm [hunk 1]
```diff
@@ -35,4 +35,7 @@
 												/datum/hallucination/delusion,
 												/datum/hallucination/oh_yeah)
 			for(var/mob/living/carbon/C in GLOB.alive_mob_list)
-				new picked_hallucination(C, TRUE)
\ No newline at end of file
+				new picked_hallucination(C, TRUE)
+		if(5)
+			for(var/i in GLOB.alive_mob_list)
+				to_chat(i, "<span class="big bold"><font color="red">Shuttle Purchase</font><br>Kate Walker has purchased Standby Evacuation Vessel "Scrapheap Challenge" for -1000 credits.</span>")
```
Comment (deathride58) [inline code/modules/events/mass_hallucination.dm]: Escape the quotes you unrobust feline

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by Trilbyspaceclone** [comment]
We also need Kate being capo 
And maybe a Hairly joining as capo

**Comment by Dankseed** [comment]
Much needed feature for some time now, glad to see it finally being added. +1 merge when

**Comment by zombieme7** [comment]
+1

**Review by deathride58 — CHANGES_REQUESTED** [review]
Make it compile

**Pull request closed without merging**